### DISCUSSION TITLE
How to add combos to miryoku?

### DISCUSSION THREAD

**Discussion by aradbm** [manna-harbour/miryoku#289] [Q&A] [answered]
Hi!
I am using miryoku on Piantor, and I want to add some combos of copy/paste and such on my left hand. How can I do that?

**Comment by manna-harbour** [comment, marked as answer]
For single handed clipboard you can use the [Button layer](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#button).

Support for custom combos will be added in future, see https://github.com/manna-harbour/miryoku/issues/94.

For now, it's the same situation as described at https://github.com/manna-harbour/miryoku/discussions/279#discussioncomment-8152849. As combos are already in use when thumb combos are enabled, they'll need to be integrated with the conditional logic.
